### ISSUE TITLE
Run time for extracting images.

### ISSUE THREAD

**Issue by jasonsheinkopf** [ddshan/hand_object_detector#33]
I used demo.py to process a set of 54 test images. On my CPU the process takes about 2 seconds per image. On my a40, for some reason it takes 8 seconds per image.

I'm only interested in the detection values not the image generation. Is this a normal time for extraction per image? I was curious if there the rate should be faster and maybe I'm doing something wrong, as it would take about 14 months to process SomethingsomethingV2 dataset at this rate.

**Comment by ddshan** [comment]
8 seconds/image is an abnormal time for processing an image. The demo code should be reasonably fast.
Could you double-check the code is running on GPU?

**Comment by jasonsheinkopf** [comment]
I agree. Would you be able to share a reasonable processing time for a single image? Should it be more like:
1 s, 0.1 s, 0.01s, etc?

**Comment by TranThanhTuan2509** [comment]
Sorry, could you please share your revised code on GitHub? I have the some problem with the original code; there are some deprecated libraries, so I need to rebuild it. However, there are still some mistakes.
